CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ CHANGELOG
 
 [//]: # (comment: Don't forget to update src/DogStatsd.php:DogStatsd::version when releasing a new version)
 
+# 1.6.0 / 2023-05-10
+
+* Add support for [Unified service tagging](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes)
+* Remove `lib-curl` dependency and events API support
```

Comment:
Can we add the specific calls that were deprecated? Just to help future googlers find this when they run `update` on their dependencies and wonder why they now have errors

Reply:
It doesn't actually remove a method from the API surface. Just the support for sending events over the API when an `APP_KEY` is present. So maybe "removed support for `APP_KEY`"?

Reply:
Just for my understanding, the case that could break here is a customer was running this without an agent connection, instead connecting straight to the API.
So before they were specifying APP_KEY and their events were correctly ingested.

After updating, this will fail silently if I understand correctly?

Reply:
Yep that's correct.

Reply:
Got it, something along these lines would be my proposal then
```suggestion
* Remove `lib-curl` dependency and support for events API (using `APP_KEY`). Submitting events now requires a connection to a Datadog Agent.
```

Reply:
Thanks, that's much better